<launch>

  <arg name="image_raw_topic" default="/ackermann_vehicle/camera/rgb/image_raw" />

  <node name="ipm_ros" pkg="ipm" type="ipm_optimized_ros.py" output="screen" required="true" >
    <param name="image_raw_topic" value="$(arg image_raw_topic)" />
  </node>

</launch>
